airbyte-integrations/connectors/destination-oracle/src/main/resources/spec.json

```diff
@@ -0,0 +1,110 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/postgres",
+  "supportsIncremental": true,
+  "supported_destination_sync_modes": ["overwrite", "append"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Postgres Destination Spec",
+    "type": "object",
+    "required": ["host", "port", "username", "database", "schema"],
+    "additionalProperties": false,
+    "properties": {
+      "host": {
+        "title": "Host",
+        "description": "Hostname of the database.",
+        "type": "string",
+        "order": 0
+      },
+      "port": {
+        "title": "Port",
+        "description": "Port of the database.",
+        "type": "integer",
+        "minimum": 0,
+        "maximum": 65536,
+        "default": 5432,
+        "examples": ["5432"],
+        "order": 1
+      },
+      "database": {
+        "title": "DB Name",
+        "description": "Name of the database.",
+        "type": "string",
+        "order": 2
+      },
+      "schema": {
+        "title": "Default Schema",
+        "description": "The default schema tables are written to if the source does not specify a namespace. The usual value for this field is \"public\".",
+        "type": "string",
+        "examples": ["public"],
+        "default": "public",
+        "order": 3
+      },
```

Comment:
https://dba.stackexchange.com/questions/37012/difference-between-database-vs-user-vs-schema#:~:text=In%20Oracle%2C%20users%20and%20schemas,that%20belong%20to%20that%20account.

basically you dont need schema... or could be a good thing if on the docs we suggest creating a user airbyte and give permission to CREATE in another user/schema =)

Reply:
Apparently in Oracle, a "user" is what a "schema" is in other SQL databases.  Oracle documentation even talks about user names in identifiers as a "schema name" in various places.  I've set this up to use users as schemas, and all the tests passed that way.  We should be good just keeping the Airbyte concept of schemas and mapping them onto Users under the hood.

Reply:
2 questions:
* How should we support namespaces? should we just not support them for oracle? (@marcosmarxm - would love your opinion since you're the only one here who has used oracle at all) @masonwheeler for context: when the source provides a namespace (e.g. if you're replicating from the public schema in postgres , the namespace is public), then the destination tries to put data in the same schema in the destination. so the data would end up in the public schema in the destination. So I guess what I'm asking is should we support that for Oracle or force all data to go into one schema / user? I _think_ the current implementation is allow us to put data in multiple schemas but requires that the user have create schema access? If my current understanding is correct, then that seems reasonable to me.
* Does it make sense to have both of these fields? I guess the idea is that the provided username has to have permission to create multiple schemas? but the schema field is the default place we will put data if no namespace is provided. the idea is that if needed the username has permission to to create that schema?

Reply:
I think we shouldn't support them for now. If Airbyte needs to create **schemas** in Oracle this means we need to create **users**(user and passwords) and for that the **Airbyte user** must need admin privileges to do that because it needs to give permissions and manage this to every new schema.